client/src/components/exercise/lab7/Main.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/prop-types */
```

Comment:
required proptypes would also make this a little bit easier to track 
https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Resolved.